pkg/bootstrap/docker/run/run.go

```diff
@@ -215,6 +227,13 @@ func (h *Runner) startContainer(id string) error {
 }
 
 func (h *Runner) runWithOutput() (string, string, int, error) {
+	userNsMode, err := h.dockerHelper.UserNamespaceEnabled()
+	if err != nil {
+		return "", "", 0, err
+	} else if userNsMode {
+		h.hostConfig.Privileged = false
```

Comment:
> Why set this false?

Because privileged option can't be used in user namespace mode (see the error message in the original issue: `Privileged mode is incompatible with user namespaces`).

> check here if Privileged is set

Yes, thanks. I'll update this condition tomorrow.